metrics.py

```diff
@@ -0,0 +1,185 @@
+# Copyright 2019 Google LLC.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#
+# 1. Redistributions of source code must retain the above copyright notice,
+#    this list of conditions and the following disclaimer.
+#
+# 2. Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#
+# 3. Neither the name of the copyright holder nor the names of its
+#    contributors may be used to endorse or promote products derived from this
+#    software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+"""Used to collect anonymous DeepVariant metrics."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import atexit
+import functools
+import json
+import logging
+import time
+import uuid
+
+import requests
+from typing import Dict, Optional, Text
+
+_CLEARCUT_ENDPOINT = 'https://play.googleapis.com/log'
+_CLOUD_HCLS = 'CLOUD_HCLS'
+_CONCORD = 'CONCORD'
+_DEEP_VARIANT_RUN = 'DeepVariantRun'
+_HTTP_REQUEST_TIMEOUT_SEC = 10
+_PYTHON = 'PYTHON'
+_VIRTUAL_CHC_DEEPVARIANT = 'virtual.chc.deepvariant'
```

Comment:
Is this name set in stone already?  otherwise virtual.hcls.deepvariant would be preferable

Reply:
I am not really sure if changing this will affect anything in a bad way. I am running tests now and will update this comments as soon as I find out more.